data/pug/pug.txt

```diff
@@ -168,6 +168,33 @@ phrase "hostile pug"
 	word
 		"."
 
+outfit "Pug Jump Drive"
+	category Systems
+	cost 1000000
+	thumbnail "outfit/jump drive"
+	"mass" 20
+	"outfit space" -20
+	"jump drive" 1
+	"jump speed" 0.3
+	"jump fuel" 200
+	description "The workings of the jump drive are a total mystery. The best that human scientists have been able to find out is that this device is a containment field for matter in a state that cannot normally exist in this universe; if the drive is opened up, the field collapses and its contents disappear. Therefore, even though a few jump drives have been captured over the years, it has so far proved impossible to reverse engineer them."
+	description "The Pug Jump Drive is between the Korath and Quarg in strength and worth."
```

Comment:
I don't think this is a good description, especially since in a Checkmate playthrough you may well not have heard of the Korath at this point, much less be aware that they have Jump Drives.

Reply:
I removed that second paragraph. I hope that's enough. Thanks!